docusaurus/docs/how-to-guides/data-source-plugins/data-source-configuration.md

```diff
@@ -0,0 +1,137 @@
+---
+id: data-source-configuration
+title: Configure data sources
+description: A guide to data source configuration.
+keywords:
+  - grafana
+  - data source plugin
+  - datasource
+  - data source plugin configuration
+---
+
+# Best practices for data source plugin configuration
+
+To deliver a consistent and user-friendly experience to Grafana users, we recommend that Grafana plugin developers follow uniform guidelines when building data source plugins. This guide outlines recommended best practices to help you achieve this goal.
+
+## Logos
+
+If your plugin requires a logo, follow these recommendations:
+
+- Ensure you obtain permission from the trademark holder.
+- Follow the brand guidelines, typically available in a company’s media kit or press kit.
+- Use logos with a transparent background, avoiding white backgrounds.
+- Logos should be in SVG format.
+
+**Example:** [MongoDB](https://www.mongodb.com/company/newsroom/brand-resources)  
+**Example:** [Datadog](https://www.datadoghq.com/about/resources/)
+
+## Description and required fields
+
+We recommend using the `DataSourceDescription` component from the [@grafana/plugin-ui](https://github.com/grafana/plugin-ui) repository. The plugin name and documentation link are required, but details about other fields is optional. The plugin name and documentation link are required, but details about other fields is optional.
+
+For required fields, avoid manually typing `*`. Instead, pass the `required` prop to the `InlineField` component. This prop automatically adds a `*` to the label.
+
+**Example of DataSourceDescription component:**
+
+![Description](/img/data-source-config/config-data-source-description.png)
+
+**Example of required field:**
+
+![Connection](/img/data-source-config/config-connection.png)
+
+## Sections
+
+Divide the data source configuration page into multiple sections using the `ConfigSection` component.
+
+- The `section` description property is optional but recommended if the section contains more than one field.
+- For sections with optional or rarely used settings, it's best to keep them collapsed by default.
+- Use the `ConfigSubSection` component for organizing subsections within a section. Avoid separating subsections with horizontal lines.
+
+**Example of sections:**
+
+![Sections](/img/data-source-config/config-sections.png)
+
+## Authentication component
+
+Use the `Authentication` component for handling authentication on the configuration page. This component is already wrapped inside a `ConfigSection`, so additional wrapping is unnecessary.
```

Comment:
Let's remove all calls to use `Authentication`  and keep only `DataSourceHttpSettings` without indicating that is deprecated (it is not deprecated)